week2/homework/src/index.js

```diff
@@ -1,3 +1,97 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+const fileName = `${__dirname}/todoList.txt`;
+
+const command = process.argv[2];
+const args = process.argv.slice(3);
+
+const newItem = process.argv.slice(4);
```

Comment:
Variables `args` and `newItem` are not used in all commands, so better to declare them when needed only.